src/react/JSONTree/obj-type.js

```diff
@@ -1,3 +1,7 @@
 export default function(obj) {
-  return Object.prototype.toString.call(obj).slice(8, -1);
+  const type = Object.prototype.toString.call(obj).slice(8, -1);
+  if (type === 'Object' && typeof obj[Symbol.iterator] === 'function') {
```

Comment:
I am not sure this is correct approach. There might be even more iterable types returning its own toString representation. 

Originally I had there just check for iterator existence, but that was wrong for String which implements iterator for its characters. Similar goes for the Array.

It might be safer bet to just check `typeof obj === 'object'` and excluding array from it. Any other ideas?